ui/apps/platform/src/Containers/Compliance/Dashboard/ComplianceScanProgress.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import {
+    Card,
+    CardBody,
+    CardTitle,
+    Progress,
+    ProgressMeasureLocation,
+} from '@patternfly/react-core';
+
+import { ComplianceRunStatusResponse } from './useComplianceRunStatuses';
+
+export type ComplianceDashboardCurrentProps = {
+    runs: ComplianceRunStatusResponse['complianceRunStatuses']['runs'];
+};
+
+function ComplianceScanProgress({ runs }: ComplianceDashboardCurrentProps) {
+    const unfinishedRunCount = runs.filter((run) => run.state !== 'FINISHED').length;
+    const finishedRunCount = runs.length - unfinishedRunCount;
+
+    const title =
+        unfinishedRunCount === 0
+            ? 'Compliance scanning complete'
+            : 'Compliance scanning in progress';
+
+    return (
+        <Card>
+            <CardTitle id="compliance-scan-progress-title">{title}</CardTitle>
+            <CardBody>
+                <Progress
+                    aria-labelledby="compliance-scan-progress-title"
+                    size="sm"
+                    variant={unfinishedRunCount === 0 ? 'success' : undefined}
+                    value={finishedRunCount}
+                    min={0}
+                    max={runs.length}
+                    measureLocation={ProgressMeasureLocation.outside}
```

Comment:
nit: I thought the UI decided that we didn't need to import the PF constants into TS components, because TS type checking guarded against us using an unsupported constant.

Reply:
TypeScript string enumeration is the source of truth:
https://github.com/patternfly/patternfly-react/blob/main/packages/react-core/src/components/Progress/Progress.tsx#L20

By the way, PatternFly team explained the current process to generate docs for components which depends on inline declaration is a reason why these have not yet been exported as a type. Although we do not need an explicit type here.